recipes-browser/webkitgtk/webkitgtk_2.40.0.bb

```diff
@@ -0,0 +1,131 @@
+SUMMARY = "WebKit web rendering engine for the GTK+ platform"
```

Comment:
this recipe is basically a copy from ./recipes-browser/webkitgtk/webkitgtk_2.38.5.bb   . I'd suggest to remove the ./recipes-browser/webkitgtk/webkitgtk_2.38.5.bb recipe. Otherwise, if there is a strong reason to keep it, then move the shared code inside a `webkitgtk-common.inc` file

Reply:
Since 2.40 is very recent and 2.38 still maintained and delivering bufixes, I thought that it would be good to keep 2.38 for a few months.

Also there are jobs even 2.36 (WPE, not GTK, but still).

What's the criteria?

(To be clear, I am fine with the plans, just trying to learn the logic to save work in the future).